src/emulator/auth/operations.ts

```diff
@@ -1460,7 +1460,43 @@ function signInWithIdp(
     }
   }
 
-  let { response, rawId } = fakeFetchUserInfoFromIdp(providerId, claims);
+  if (reqBody.pendingToken) {
```

Comment:
I don't think Auth Emulator even generates `pendingToken`s at all. Would you mind double checking this before implementing parsing logic?